src/signals/incident-management/containers/IncidentSplitContainer/index.test.js

```diff
@@ -20,15 +20,14 @@ store.dispatch(requestIncidentSuccess(incidentJson));
 store.dispatch(fetchCategoriesSuccess(categoriesPrivate));
 
 const incidentModel = makeSelectIncidentModel(store.getState());
-
 jest.spyOn(reactRouterDom, 'useParams').mockImplementation(() => ({
   id: '42',
 }));
 
-jest.mock('models/incident/selectors', () =>
-  // eslint-disable-next-line global-require
-  jest.fn(() => require('utils/__tests__/fixtures/incident.json'))
-);
+// jest.mock('models/incident/selectors', () =>
+//   // eslint-disable-next-line global-require
+//   jest.fn(() => require('utils/__tests__/fixtures/incident.json'))
+// );
```

Comment:
Shall we remove those commented lines?

Reply:
Totes. Already did that, but forgot to push the change.